codespell_lib/data/dictionary.txt

```diff
@@ -25272,6 +25273,7 @@ freez->frees, freeze,
 freezs->freezes
 freind->friend
 freindly->friendly
+freit->freight
```

Comment:
Same here, it's also a verb so could you add all verbal forms?
```suggestion
freit->freight
freited->freighted
freiting->freighting
freits->freights
```

Reply:
Thank you, DimitriPapadopoulos.
I've acknowledged all suggestions, I'll make necessary adjustment